docs-2.0/3.ngql-guide/3.data-types/8.map.md

```diff
@@ -15,6 +13,8 @@ nebula> YIELD {key: 'Value', listKey: [{inner: 'Map1'}, {inner: 'Map2'}]}
 +-------------------------------------------------------------+
 ```
 
-## Map projection
+## OpenCypher compatibility
+
+- A composite data type (i.e. map, and list) cannot be stored as (vertex's or edge's) properties.
```

Comment:
Plz keep consistent: 
A composite data type (i.e. set, map, and list) CANNOT be stored as properties of vertices or edges.